src/Commands/App/Builder.php

```diff
@@ -100,8 +100,16 @@ protected function compile(string $name): Builder
             ->getCompiler($name);
 
         $structure = config('app.structure') ?: $this->structure;
+        
+        // See - https://github.com/laravel-zero/framework/pull/102
+        $regex = '#'.implode('|', $structure).'#';
 
-        $compiler->buildFromDirectory(BASE_PATH, '#'.implode('|', $structure).'#');
+        if (stristr(PHP_OS, "WINNT") != FALSE) {
+            $compiler->buildFromDirectory(BASE_PATH, str_replace('\\', '/', $regex));
```

Comment:
I think `preg_quote()` would be a better fit here, there might be some other unforeseen cases that won't be solved by just replacing `\` with `/`.

Reply:
i was pulled down your commit, it seems ok  but unfortunately compiled laravel zero application does't running on windows platform.

this PR is build and running correctly on windows. thanks